feature/gnoi/packet_link_qualification/tests/packet_link_qualification_test/packet_link_qualification_test.go

```diff
@@ -256,36 +278,74 @@ func TestLinkQualification(t *testing.T) {
 	}
 	t.Logf("generatorCreateRequest: %v", generatorCreateRequest)
 
-	reflectorCreateRequest := &plqpb.CreateRequest{
-		Interfaces: []*plqpb.QualificationConfiguration{
-			{
-				Id:            plqID,
-				InterfaceName: dp2.Name(),
-				EndpointType: &plqpb.QualificationConfiguration_PmdLoopback{
-					PmdLoopback: &plqpb.PmdLoopbackConfiguration{},
-				},
-				Timing: &plqpb.QualificationConfiguration_Rpc{
-					Rpc: &plqpb.RPCSyncedTiming{
-						Duration: &durationpb.Duration{
-							Seconds: int64(plqDuration.testDuration.Seconds()),
-						},
-						PreSyncDuration: &durationpb.Duration{
-							Seconds: int64(plqDuration.preSyncDuration.Seconds()),
-						},
-						SetupDuration: &durationpb.Duration{
-							Seconds: int64(plqDuration.setupDuration.Seconds()),
-						},
-						PostSyncDuration: &durationpb.Duration{
-							Seconds: int64(plqDuration.postSyncDuration.Seconds()),
+	var reflectorCreateRequest *plqpb.CreateRequest
+	switch dut2.Vendor() {
```

Comment:
Are we switching on the vendor without a deviation? Is there a discussion somewhere that says both types of requests are specification-compliant or are we awaiting resolution as to which one satisfies our expected contract?

Reply:
Both types of requests are specification-compliant. Please refer to : [https://github.com/openconfig/gnoi/blob/main/packet_link_qualification/index.md#the-service-implementation-must-support-multiple-generations-of-forwarding-hardware-which-have-different-capabilities-with-regards-to-both-generation-and-reflection-of-packets](url)